src/main/webapp/js/studentProfile.js

```diff
@@ -36,6 +34,22 @@ $(function() {
             $('#profilePicEditRotateRight').click(function() {
                  picture.guillotine('rotateRight');
             });
+            $('#profilePicEditPanUp').click(function() {
+                var data = picture.guillotine('getData');
+                picture.guillotine('instance')._offset(data.x / data.w, (data.y - 10) / data.h);
+            });
+            $('#profilePicEditPanLeft').click(function() {
+                var data = picture.guillotine('getData');
+                picture.guillotine('instance')._offset((data.x - 10) / data.w, data.y / data.h);
+            });
+            $('#profilePicEditPanRight').click(function() {
+                var data = picture.guillotine('getData');
+                picture.guillotine('instance')._offset((data.x + 10) / data.w, data.y / data.h);
+            });
+            $('#profilePicEditPanDown').click(function() {
+                var data = picture.guillotine('getData');
+                picture.guillotine('instance')._offset(data.x / data.w, (data.y + 10) / data.h);
```

Comment:
just one last thing, the calculation for this is not straight forward. if it is simple, could you explain it here? or if it needs long explanation, could you provide a link to a resource?